jujugui/templates/index.html.go

```diff
@@ -49,6 +49,9 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
          below.
     -->
     <script src="{{.configURL}}"></script>
+	<!-- Load the version file to be displayed in the GUI.
```

Comment:
As you saw with the test changes, everything kind of falls over if that value isn't defined. Could you add to one of the preceding script blocks a default value for window.GUI_VERSION in both of the index files to avoid any issues if the file doesn't load?